### ISSUE TITLE
Missing table in export

### ISSUE THREAD

**Issue by youthelectronix** [BCcampus/pressbooks-mpdf#71]
### Description 

Table is not appearing in an mPDF export.

**Pressbooks version**
<img width="982" alt="screen shot 2018-11-15 at 12 57 06 pm" src="https://user-images.githubusercontent.com/45080034/48579839-989a2e80-e8eb-11e8-8742-ae16e42d78f4.png">

**text editor**
<img width="1708" alt="screen shot 2018-11-15 at 3 34 38 pm" src="https://user-images.githubusercontent.com/45080034/48580051-19592a80-e8ec-11e8-99a5-5b0e2039a033.png">

**mPDF version**
<img width="761" alt="screen shot 2018-11-15 at 12 56 44 pm" src="https://user-images.githubusercontent.com/45080034/48579874-ab146800-e8eb-11e8-9ab2-f6bc77ad0a1a.png">

### System Information

- mPDF for Pressbooks: 3.2.1 
- Pressbooks: 5.5.6
- WordPress: 4.9.8

**Comment by bdolor** [comment]
thank you @youthelectronix - will take a look!

**Comment by josieg** [comment]
**TLDR:** I think the problem is in the attributes/values within the `<Table>` element that @youthelectronix is using. Tables created with the current default Pressbooks markup export no problem in mPDF. And when I removed  `style="width: auto;"` from the table element, I was able to export the table in mPDF.

So I don't know if this is a bug or if the table just needs to be recreated using the updated Pressbooks markup.  Here are the specifics of what I found:

### When I used the following table markup, which is the deafult table markup I get when I create a new table in Pressbooks, I have no problems exporting a table in mPDF:
**HTML view:**
![mpdftablenormalmarkup](https://user-images.githubusercontent.com/23706762/48640374-59bbb580-e98b-11e8-8cff-b4e535f922a9.JPG)
**mPDF view:**
![mpdftablenormalmarkup1](https://user-images.githubusercontent.com/23706762/48640443-98ea0680-e98b-11e8-985a-a0d987ae3c9b.JPG)

### However, when I imitate the html provided in the above example, I get the following:
**HTML view:**
![mpdftableabnormalmarkup](https://user-images.githubusercontent.com/23706762/48640555-dbabde80-e98b-11e8-9e12-9d45a06d741a.JPG)
**Visual Editor view:**
![mpdftableabnormalmarkupve](https://user-images.githubusercontent.com/23706762/48640687-3f360c00-e98c-11e8-8fca-46f685efdccb.JPG)
**MPDF view:**
![mpdftableabnormalmarkup1](https://user-images.githubusercontent.com/23706762/48640592-fc743400-e98b-11e8-828a-b12cefefe8e9.JPG)
**Note that I also tried creating the table using the same width="XXpx" values used in the previous example, but that did not affect the mPDF output.

### When I removed `style="width: auto;"` from the table element, I get the following:
**HTML view:**
![mpdftablesimplemarkup](https://user-images.githubusercontent.com/23706762/48641474-bec4da80-e98e-11e8-93df-20ee51a4278f.JPG)
**mPDF view:**
![mpdftablesimplemarkup1](https://user-images.githubusercontent.com/23706762/48641502-d7cd8b80-e98e-11e8-8c86-215d69e1da16.JPG)

**Comment by josieg** [comment]
And I forgot to compare behaviour to Prince.

The `style="width: auto;"` is not a problem in Prince PDF exports. Table elements with that attribute/value appear fine, so this is definately a mPDF problem.
